src/System.Management.Automation/resources/TabCompletionStrings.resx

```diff
@@ -329,6 +329,218 @@
   <data name="shrOperatorDescription" xml:space="preserve">
     <value>Shift Right bit operator. Inserts zero in the left-most bit position. For signed values, sign bit is preserved.</value>
   </data>
+  <data name="NameHashtableKeyDescription" xml:space="preserve">
+    <value>[string]
+Specifies the name of the property being created.</value>
+  </data>
+  <data name="LabelHashtableKeyDescription" xml:space="preserve">
+    <value>[string]
+Specifies the name of the property being created.</value>
```

Comment:
Maybe "the name" -> "the label" or "the name label"

Reply:
I think "Name" is clearer than both alternatives.